src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs

```diff
@@ -1,24 +1,58 @@
-﻿using LT.DigitalOffice.Kernel.Configurations;
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.File;
+using LT.DigitalOffice.Models.Broker.Requests.Message;
+using LT.DigitalOffice.Models.Broker.Requests.Project;
+using LT.DigitalOffice.Models.Broker.Requests.Token;
+using LT.DigitalOffice.Models.Broker.Requests.User;
 
 namespace LT.DigitalOffice.UserService.Models.Dto.Configurations
 {
     public class RabbitMqConfig : BaseRabbitMqConfig
```

Comment:
here i guess you have extra injects, check which requests used in consumers and don't inject them

Reply:
Fixed